src/test/resources/db/migration/V904__add_step_actors_data.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO step_actors (id, step_uuid, actor, actor_option_id) VALUES (1, '71793b64-545e-4ae7-9936-610639093857', 'Actor name', 1);
```

Comment:
don't set the value of a serial field, `id` in this case - postgres will do this automatically

Reply:
updated